src/parse/parser.rs

```diff
@@ -1435,8 +1424,8 @@ impl<'help, 'app> Parser<'help, 'app> {
                 }
             }
         } else if let Some(fv) = attached_value {
-            let v = fv.trim_start_n_matches(1, b'=');
-            debug!("Found - {:?}, len: {}", v, v.len());
+            let v = fv.strip_prefix("=").unwrap_or(fv);
```

Comment:
Can this be an `unwrap`? The original code was unclear about if this "=" is required.